edpop_explorer/readers/kvcs.py

```diff
@@ -0,0 +1,79 @@
+import csv
+from pathlib import Path
+from typing import List
+from edpop_explorer import Reader, ReaderError, BiographicalRecord
+from rdflib import URIRef
+
+
+class KVCSReader(Reader):
+    """ KVCS database reader. Access with command 'kvcs'."""
+    FILENAME = Path(__file__).parent / 'data' / 'biblio_kvcs.csv'
+    CATALOG_URIREF = URIRef(
+        'https://edpop.hum.uu.nl/readers/kvcs'
+    )
+    IRI_PREFIX = "https://edpop.hum.uu.nl/readers/kvcs/"
+    FETCH_ALL_AT_ONCE = True
+    SHORT_NAME = "KVCS"
+    DESCRIPTION = "Drukkers & Uitgevers in KVCS"
```

Comment:
Add `READERTYPE = BIOGRAPHICAL` (this is not done consistently in all readers -- my bad)